charts/foundry/values.yaml

```diff
@@ -0,0 +1,109 @@
+# Default values for foundry.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+anvil:
+  host: '0.0.0.0'
+  port: '8545'
+  blockTime: 1
+  forkURL: 'https://goerli.infura.io/v3/...'
+  forkBlockNumber: "10448829"
+  forkRetries: "5"
+  forkTimeout: "45000"
+  forkComputeUnitsPerSecond: "330"
+#  forkNoRateLimit: "true"
```

Comment:
will that default to false if it's commented out?

Reply:
Yep